legacy/coverage/coverageEndpoint.js

```diff
@@ -1,11 +1,15 @@
 var express = require('express');
 var router = express.Router();
 
-var coverageEndpoint = function(coverage, optionalCoverage) {
+var coverageEndpoint = function(coverage, azurecoverage, optionalCoverage) {
     router.get('/', function(req, res, next) {
         res.status(200).send(Object.assign({}, coverage, optionalCoverage)).end();
     });
 
+    router.get('/azure', function(req, res, next) {
+        res.status(200).send(Object.assign({}, azurecoverage)).end();
```

Comment:
Can you please merge azure coverage into the main endpoint. I don't think we care about the difference on .net side.